fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -144,29 +144,71 @@ def self.install_missing_wwdr_certificates
       # Install all Worldwide Developer Relations Intermediate Certificates listed here: https://www.apple.com/certificateauthority/
       missing = WWDRCA_CERTIFICATES.map { |c| c[:alias] } - installed_wwdr_certificates
       missing.each do |cert_alias|
-        install_wwdr_certificate(cert_alias)
+        url = WWDRCA_CERTIFICATES.find { |c| c[:alias] == cert_alias }.fetch(:url)
+        file = Tempfile.new(File.basename(url))
+
+        unless fetch_certificate(url, file)
+          UI.verbose("Could not fetch certificate #{cert_alias}")
+          next
+        end
```

Comment:
I think it would make more sense to have `fetch_certificate(url)` determine the name of the temp destination file and return that as a return value, rather than pass it as parameter, wdyt?

Reply:
Yeah, good idea!